### PR TITLE
fix(calendar): set today disabled date style

### PR DESCRIPTION
**Pull request by radireddy**
make today disabled day style same as any other day disabled

ref: UK-194

Signed-off-by: Ravindra Adireddy <ravindra.adireddy@ovhcloud.com>

### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

## Title of the Pull Requests 
OUI Calendar today date style

### Description of the Change
make today disabled day style same as any other day disabled

### Benefits
Improved customer experiance

### Possible Drawbacks
none

### Applicable Issues
UK-194

### CODE REVIEW

**Review by anooparveti — DISMISSED** [review]

**Review by JeremyDec — COMMENTED** [review]
Hello @radireddy,

Changes LGTM, but could you please amend your commit message to follow the format:

`type(scope): present tense verb + action`

So for example: `fix(oui-calendar): set today class as flatpickr class direct child `.

Thanks

**Comment by radireddy** [comment]
> Hello @radireddy,
> 
> Changes LGTM, but could you please amend your commit message to follow the format:
> 
> `type(scope): present tense verb + action`
> 
> So for example: `fix(oui-calendar): set today class as flatpickr class direct child `.
> 
> Thanks

Thanks. I have changed the commit message

**Review by anooparveti — DISMISSED** [review]

File: packages/components/calendar/src/less/flatpickr.less [hunk 1]
```diff
@@ -296,11 +296,6 @@
     }
 
     &.today {
```
[the remark this answers is not in the document]
Comment (radireddy) [reply packages/components/calendar/src/less/flatpickr.less:298]: I am trying to apply this style for disabled date, exactly opposite to what you have suggested.

[the remark this answers is not in the document]
Comment (ghyenne) [reply packages/components/calendar/src/less/flatpickr.less:298]: I suggested adding this to the original code and not changing anything else.

**Review by ghyenne — APPROVED** [review]

**Review by mohammed-zahaf — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

**Pull request merged**